scrapers/vixenNetwork.py

```diff
@@ -0,0 +1,98 @@
+import json
+import os
+import re
+import requests
+import sys
+
+
+def log(*s):
+    print(*s, file=sys.stderr)
+
+
+def get_from_url(url_to_parse):
+    m = re.match(r'(?:https?://)?(?:www\.)?((\w+)\.com)/([a-z-]+)', url_to_parse)
```

Comment:
`([a-z-]+)`
thats very exclusive
scenes with numbers fail eg `part-1` , `https://www.tushy.com/eva-part-1`

Reply:
Ahh... That's what you call an oversight.